components/Menu/styles/styles.tsx

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 import { BackGroundColor, FontColor, GridBreakpoints } from 'style/Theme'
-import { StyledLink } from './util'
+import { StyledLink } from './StyledLink'
```

Comment:
StyledLink should be outside of `style` folder since its a independent component. Actually, I think we should consider move this completely outside as a `common component` IF we plan to reuse this component in future 🤔